### PR TITLE
Error page added

### PR DESCRIPTION
**Pull request by danial-rokt**
Simulator Screen Shot - iPhone 12 Pro - 2021-07-19 at 15 38 59

### CODE REVIEW

File: RoktDemo.xcworkspace/xcuserdata/danial.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist [hunk 1]
```diff
@@ -3,4 +3,22 @@
    uuid = "F886A9A5-9837-4F02-92A8-944C528CF2F5"
    type = "0"
    version = "2.0">
+   <Breakpoints>
```
Comment (DJingSeo) [inline RoktDemo.xcworkspace/xcuserdata/danial.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:6]: I don't believe you want to add breakpoint =)

File: RoktDemo/ViewModels/ErrorViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    case general
+}
+
+struct ErrorViewModel {
+    let error: AFError?
+    
+    func getErrorMessageType() -> ErrorMessageType {
+        if error != nil && error!.isResponseSerializationError {
+            return.general
+        }
```
Comment (DJingSeo) [inline RoktDemo/ViewModels/ErrorViewModel.swift:26]: Shouldn't it be general when error is null ?

Comment (danial-rokt) [reply RoktDemo/ViewModels/ErrorViewModel.swift:29]: done

File: RoktDemo.xcworkspace/xcuserdata/danial.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist (hunk 1 above)
Comment (danial-rokt) [reply RoktDemo.xcworkspace/xcuserdata/danial.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist]: Removed

**Review by DJingSeo — APPROVED** [review]

**Pull request merged**